### DISCUSSION TITLE
How to view span events in the traces viewer

### DISCUSSION THREAD

**Discussion by BohdanPetryshyn** [grafana/clickhouse-datasource#1000] [Q&A]
Looks like span events are not visualized in Clickhouse datasource's trace viewer. According to [this discussion](https://github.com/grafana/clickhouse-datasource/discussions/335), this is a limitation of the Grafana trace view.

**Question:** Is there any workaround to visualize the span events as part of the traces view without falling back to the table view?

For reference, this is how span events look like in Tempo datasource:

<img width="741" alt="image" src="https://github.com/user-attachments/assets/c341d294-bdec-41c3-81cf-960c2f49ef71">

**Comment by SpencerTorres** [comment]
Similar issue, we need a way to load these from ClickHouse: https://github.com/grafana/clickhouse-datasource/issues/977
